### ISSUE TITLE
The connectors fails with keycloak v22 (and v23) due to 'Unrecognized field "features"'

### ISSUE THREAD

**Issue by netchild** [openstandia/connector-keycloak#16]
The trace is:

org.identityconnectors.framework.common.exceptions.ConnectorException(javax.ws.rs.client.ResponseProcessingException: javax.ws.rs.ProcessingException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "features" (class org.keycloak.representations.info.ServerInfoRepresentation), not marked as ignorable (15 known properties: "protocolMapperTypes", "providers", "identityProviders", "themes", "passwordPolicies", "clientInstallations", "memoryInfo", "enums", "socialProviders", "clientImporters", "profileInfo", "componentTypes", "systemInfo", "cryptoInfo", "builtinProtocolMappers"])? at [Source: (org.jboss.resteasy.specimpl.AbstractBuiltResponse$InputStreamWrapper); line: 1, column: 1184] (through reference chain: org.keycloak.representations.info.ServerInfoRepresentation["features"]))

**Comment by netchild** [comment]
If I simply change the dependency from version 20.0.x to 23.0.1 in pom.xml, I get this:
```
Bad type on operand stack
Exception Details:
  Location:
    jp/openstandia/connector/keycloak/rest/KeycloakAdminRESTAdminClient.<init>(Ljava/lang/String;Ljp/openstandia/connector/keycloak/KeycloakConfiguration;)V @29: invokestatic
  Reason:
    Type 'org/jboss/resteasy/core/providerfactory/ResteasyProviderFactoryImpl' (current frame, stack[0]) is not assignable to 'javax/ws/rs/ext/RuntimeDelegate'
  Current Frame:
    bci: @29
    flags: { }
    locals: { 'jp/openstandia/connector/keycloak/rest/KeycloakAdminRESTAdminClient', 'java/lang/String', 'jp/openstandia/connector/keycloak/KeycloakConfiguration', 'java/lang/RuntimeException' }
    stack: { 'org/jboss/resteasy/core/providerfactory/ResteasyProviderFactoryImpl' }
```

**Comment by wadahiro** [comment]
@netchild 

Sorry for the late reply.
I have just released v1.1.6 which should be available from the [Maven repository](https://repo1.maven.org/maven2/jp/openstandia/connector/connector-keycloak/) soon. I hope you will try this one.

https://github.com/openstandia/connector-keycloak/releases/tag/v1.1.6

**Comment by netchild** [comment]
Thanks for the new version. It connects now to keycloak, and I can see users. I haven't looked further yet, just switched over to here to report that there is no error anymore.
